### DISCUSSION TITLE
Fixed Edition Full Example

### DISCUSSION THREAD

**Discussion by JacobInCode** [metaplex-foundation/metaplex-program-library#697] [General]
Is there a full example of the fixed edition contract with a frontend anywhere? thanks.

**Comment by samuelvanderwaal** [comment]
> Is there a full example of the fixed edition contract with a frontend anywhere? thanks.

Do you mean Fixed Price Sale?

**Reply by JacobInCode** [reply]
Yes I do!

**Reply by samuelvanderwaal** [reply]
The only docs we have on it right now are these: https://docs.metaplex.com/programs/fixed-price-sale/

**Reply by JacobInCode** [reply]
Thank you - I'm working on a fixe price marketplace. Are these contracts good to use? Are there any examples of frontends built with them?
